addons/l10n_in/models/company.py

```diff
@@ -37,6 +37,77 @@ class ResCompany(models.Model):
     l10n_in_pan_type = fields.Char(string="PAN Type", compute="_compute_l10n_in_pan_type")
     l10n_in_gst_state_warning = fields.Char(related="partner_id.l10n_in_gst_state_warning")
 
+    # TDS/TCS settings
+    l10n_in_tds_feature = fields.Boolean(string="TDS", inverse="_inverse_l10n_in_tds_feature")
+    l10n_in_tcs_feature = fields.Boolean(string="TCS", inverse="_inverse_l10n_in_tcs_feature")
+    l10n_in_withholding_account_id = fields.Many2one(
+        comodel_name='account.account',
+        string="TDS Account",
+        check_company=True,
+    )
+    l10n_in_withholding_journal_id = fields.Many2one(
+        comodel_name='account.journal',
+        string="TDS Journal",
+        check_company=True,
+    )
+    l10n_in_tan = fields.Char(string="TAN", help="Tax Deduction and Collection Account Number")
+
+    # GST settings
+    l10n_in_is_gst_registered = fields.Boolean(string="Registered Under GST", inverse="_inverse_l10n_in_is_gst_registered")
+    l10n_in_gstin_status_feature = fields.Boolean(string="Check GST Number Status")
+    l10n_in_gst_efiling_feature = fields.Boolean(string="GST E-Filing & Matching Feature")
+    l10n_in_fetch_vendor_edi_feature = fields.Boolean(string="Fetch Vendor E-Invoiced Document")
+
+    # E-Invoice
+    l10n_in_edi_feature = fields.Boolean(string="E-Invoicing")
+    # E-Waybill
+    l10n_in_ewaybill_feature = fields.Boolean(string="E-Way bill")
+    # ENet Batch Payment
+    l10n_in_enet_vendor_batch_payment_feature = fields.Boolean(string="ENet Vendor Batch Payment")
+
+    def _inverse_l10n_in_tds_feature(self):
+        for company in self:
+            if company.l10n_in_tds_feature:
+                tds_group_id = self.env['account.chart.template'].ref('tds_group').id
+                tds_taxes = self.env['account.tax'].with_context(active_test=False).search([('tax_group_id', '=', tds_group_id)])
+                tds_taxes.write({'active': True})
+            if company.child_ids:
+                company.child_ids.write({'l10n_in_tds_feature': company.l10n_in_tds_feature})
+
+    def _inverse_l10n_in_tcs_feature(self):
+        for company in self:
+            if company.l10n_in_tcs_feature:
+                tcs_group_id = self.env['account.chart.template'].ref('tcs_group').id
+                tcs_taxes = self.env['account.tax'].with_context(active_test=False).search([('tax_group_id', '=', tcs_group_id)])
+                tcs_taxes.write({'active': True})
+            if company.child_ids:
+                company.child_ids.write({'l10n_in_tcs_feature': company.l10n_in_tcs_feature})
```

Comment:
The inverse method will only be triggered when the value of the field `l10n_in_tcs_feature` is modified and saved. However, it won't be called when the view is initially loaded. In case of a computed field, it will be executed every time the view is loaded. Therefore, using the 'inverse' method would be the better option in this case. Please correct me if I'm wrong.